profile.md

```diff
@@ -350,6 +434,25 @@ deployment:
   "space_guid": "15823972-c216-4ba5-9f3f-e75b0b891297"
   ```
 
+- `app_annotations`
+
+  Version: 2.15
```

Comment:
Probably all the versions will have to change to `2.16` now since the working group has cut a release.

Reply:
thanks @waterlink, resolved in https://github.com/openservicebrokerapi/servicebroker/pull/658/commits/80268a5fc4d73d3778cb80424424cad8e9cd64a5